contrib/upgrade.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

More portable for linux distro's and systems. [See](https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash#16365367)